kstreams-fk-join/aggregator/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,22 @@
+####
+# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
+#
+# Before building the docker image run:
+#
+# mvn package
+#
+# Then, build the image with:
+#
+# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/kafka-streams-quickstart-producer-jvm .
+#
+# Then run the container using:
+#
+# docker run -i --rm -p 8080:8080 quarkus/kafka-streams-quickstart-producer-jvm
+#
+###
+FROM fabric8/java-centos-openjdk8-jdk
```

Comment:
Maybe using a more recent Java version?

Reply:
Shouldn't matter too much for these purposes here.